rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -0,0 +1,88 @@
+# - Arguments
```

Comment:
this file was indented using spaces and tabs, please don't use spaces

Reply:
sure :+1: